<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>565</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>565</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Music Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QPushButton" name="btnGenerate">
           <property name="toolTip">
            <string extracomment="Generate a randomly generated song."/>
           </property>
           <property name="text">
            <string>Generate</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QComboBox" name="genAlgorithms"/>
           </item>
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="inputSongLength"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menutest"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
